Makefile

```
@@ -66,7 +66,7 @@ upgradebuild:
 
 .PHONY: package_nlp_primitives
 package_nlp_primitives: upgradepip upgradebuild
-	python setup.py sdist
-	$(eval PACKAGE=$(shell python setup.py --version))
+	python -m build
+	$(eval PACKAGE=$(shell python -c "from pep517.meta import load; metadata = load('.'); print(metadata.version)"))
```

Is there a reason we are using that versus this? 
```suggestion
	$(eval PACKAGE := $(shell grep '__version__\s=' nlp_primitives/version.py | grep -o '[^ ]*$$'))
```

---

Alright